aoc/src/main/scala/aoc2015/Day01_2015.scala

```diff
@@ -0,0 +1,39 @@
+object Day01_2015 extends App:
+  import scala.io.Source
+  import scala.collection.mutable.ListBuffer
+
+  val lines = Source.fromFile("aoc\\src\\resources\\aoc2015\\Day01_2015").getLines.mkString
+
+  def findfloor(current: Int, instructions: String): Int = {
```

Comment:
curly brackets are not required in scala 3 code - indented code blocks the style of preference - please remove